packages/paste-website/src/pages/components/ai-chat-log/index.mdx

```diff
@@ -0,0 +1,340 @@
+export const meta = {
+  title: "AI Chat Log - Components",
+  package: "@twilio-paste/ai-chat-log",
+  description:
+    "An AI Chat Log is a collection of AI Chat components for displaying conversations between a human and an AI bot.",
+  slug: "/components/ai-chat-log/",
+};
+
+import { uid } from "@twilio-paste/uid-library";
+import { HelpText } from "@twilio-paste/help-text";
+import { Button } from "@twilio-paste/button";
+import { ButtonGroup } from "@twilio-paste/button-group";
+import { Stack } from "@twilio-paste/stack";
+import {
+  AIChatLog,
+  AIChatMessage,
+  AIChatMessageAuthor,
+  AIChatMessageBody,
+  AIChatMessageLoading,
+  AIChatLogger,
+  useAIChatLogger,
+  AIChatMessageActionCard,
+  AIChatMessageActionGroup,
+} from "@twilio-paste/ai-chat-log";
+import { ThumbsUpIcon } from "@twilio-paste/icons/esm/ThumbsUpIcon";
+import { ThumbsDownIcon } from "@twilio-paste/icons/esm/ThumbsDownIcon";
+import { RefreshIcon } from "@twilio-paste/icons/esm/RefreshIcon";
+import { CopyIcon } from "@twilio-paste/icons/esm/CopyIcon";
+
+import Changelog from "@twilio-paste/ai-chat-log/CHANGELOG.md";
+
+import { SidebarCategoryRoutes } from "../../../constants";
+
+import {
+  basicBotMessage,
+  basicHumanMessage,
+  botWithFeedback,
+  botWithLoadingStopButton,
+  botWithLoading,
+  kitchenSink,
+  aiChatLoggerExample,
+  botWithBodyActions,
+} from "../../../component-examples/AIChatLogExamples";
+import packageJson from "@twilio-paste/ai-chat-log/package.json";
+import ComponentPageLayout from "../../../layouts/ComponentPageLayout";
+import { getFeature, getNavigationData } from "../../../utils/api";
+
+export default ComponentPageLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature("AI Chat Log");
+  return {
+    props: {
+      data: {
+        ...packageJson,
+        ...feature,
+      },
+      navigationData,
+      mdxHeadings,
+      pageHeaderData: {
+        categoryRoute: SidebarCategoryRoutes.COMPONENTS,
+        githubUrl: "https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/ai-chat-log",
+        storybookUrl: "/?path=/story/components-chatlog--example-chat-log",
+      },
+    },
+  };
+};
+
+<LivePreview
+  scope={{
+    AIChatLog,
+    AIChatMessage,
+    AIChatMessageAuthor,
+    AIChatMessageBody,
+  }}
+  language="jsx"
+>
+  {`<AIChatLog>
```

Comment:
question (non-blocking): just curious, nothing wrong with it, but why are you writing this example out here and importing the others?

Reply:
The first LivePreview in the chat-log index page did the same. Seeing a trend for simpler components while looking through other doc pages not being put in examples. 

I open to changing this if we want to start a best practice on it

Reply:
Makes sense, no need to change :)